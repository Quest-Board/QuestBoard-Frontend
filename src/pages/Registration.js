/*index.js*/
import React from "react";
//Functional Component
const RegistrationPage = () => {
    return (
        <div>
            <h3>Welcome to the React Router Tutorial</h3>
            <small>Registration Page</small>
        </div>
    );
};

export default RegistrationPage;