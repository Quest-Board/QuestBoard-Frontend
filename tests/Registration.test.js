import 'regenerator-runtime/runtime';
import React from 'react';
import renderer from 'react-test-renderer';
import Registration from '../src/pages/Registration';
import { BrowserRouter } from 'react-router-dom';


global.fetch = jest.fn(() => Promise.resolve());

test('Registration snapshot test', () => {
    const component = renderer.create(
        <BrowserRouter>
            <Registration />
        </BrowserRouter>
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});