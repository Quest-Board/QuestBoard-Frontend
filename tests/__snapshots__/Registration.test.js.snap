// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Registration snapshot test 1`] = `
<form
  onSubmit={[Function]}
>
  <h3>
    Sign Up
  </h3>
  <div
    className="form-group"
  >
    <label>
      Email address
    </label>
    <input
      className="form-control"
      onChange={[Function]}
      placeholder="Enter email"
      type="email"
    />
  </div>
  <div
    className="form-group"
  >
    <label>
      Password
    </label>
    <input
      className="form-control"
      onChange={[Function]}
      placeholder="Enter password"
      type="password"
    />
  </div>
  <button
    className="btn btn-primary btn-block"
    id="RegistrationButton"
    type="submit"
  >
    Sign Up
  </button>
  <p
    className="forgot-password text-right"
  >
    Already registered 
    <a
      href="/sign-in"
    >
      sign in?
    </a>
  </p>
</form>
`;
